module/userProjects/helper.js

```diff
@@ -2362,21 +2362,70 @@ module.exports = class UserProjectsHelper {
 	}
 
 	/**
-	 * List of projects.
+	 * List of user projects.
 	 * @method
 	 * @name list
+	 * @param {String} userId -UserID
+	 * @param {Number} pageNo -Page number
+	 * @param {Number} pageSize -Page size
+	 * @param {String} searchText -Search text
+	 * @param {String} language -LanguageCode
 	 * @returns {Array} List of projects.
 	 */
 
-	static list(bodyData, pageNo, pageSize, searchText, filter) {
+	static list(userId, pageNo, pageSize, searchText, language = '') {
```

Comment:
@praveenKDass please verify this helper function is not called anywhere else and passed bodyData

Reply:
No @VISHNUDAS-tunerlabs  This function not used anywhere ,verified this